src/components/bulkConcepts/container/BulkConceptsPage.jsx

```diff
@@ -110,7 +110,7 @@ export class BulkConceptsPage extends Component {
     const lastConcept = indexOfFirstConcept + currentConcepts.length;
     const firstConcept = indexOfLastConcept - 19;
     return (
-      <div className="container custom-dictionary-concepts bulk-concepts">
+      <div className="bulk-concepts">
```

Comment:
Same here.

Reply:
The same apples here. I have removed the container class since it was aligning that 'div' to the center by default which should.

Reply:
Same applies here. I have replaced container with container-fluid.